public-ui/src/app/data-browser/components/survey-answer/survey-answer-react.component.tsx

```diff
@@ -0,0 +1,255 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+
+import { ClrIcon } from 'app/utils/clr-icon';
+import { environment } from 'environments/environment';
+import { Configuration, DataBrowserApi } from 'publicGenerated/fetch';
+import * as React from 'react';
+
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { TooltipReactComponent } from 'app/data-browser/components/tooltip/tooltip-react.component';
+export const api = new DataBrowserApi(new Configuration({ basePath: environment.publicApiUrl }));
+const containerElementName = 'root';
+
+export const styleCss =
+    `
+    .survey-tbl {
+        width: 100%;
+    }
+    .survey-tbl *{
+        font-family: "GothamBook", "Arial", sans-serif,
+        font-size: 1em
+    }
+    .survey-tbl-r.survey-tbl-head {
+        /* padding-bottom: 0; */
+        background: rgba(218, 230, 237, 0.7);
+        border: #cccccc 1px solid;
+        border-bottom: none;
+    }
+    .survey-sub-tbl .survey-tbl-exp-r:last-of-type {
+        border-bottom: none;
+    }
+    .survey-row-icon {
+        height: 1.5rem;
+        width: 1.5rem;
+        color: #2691D0;
+      }
+    .sub-question-text {
+        text-align: left;
+        font-weight: bold;
+        margin-left: 1em;
+        font-family: GothamBold, Arial, Helvetica, sans-serif;
+    }
+    .survey-row-expansion {
+        border-left: #cccccc 1px solid;
+        border-right: #cccccc 1px solid;
+    }
+    .sub-table-1 .sub-question-text, .sub-table-2 .sub-question-text {
+        padding-top: 1rem;
+    }
+    .survey-sub-tbl {
+        border-bottom: #cccccc 1px solid;
+    }
+    .survey-sub-tbl .survey-tbl-exp-r:last-of-type {
+        border-bottom: none;
+    }
+
+    .survey-sub-table {
+        margin-left: 2%;
+        width: 98%;
+    }
+
+    .survey-sub-table-2 {
+        padding-top: 1rem;
+        margin-left: 2%;
+        width: 98%;
+    }`;
+
+
+interface SurveyRowProps {
+    // survey_concept_id
+    stratum1: number;
+    // question_concept_id
+    stratum2: number;
+    // answer_concept_id
+    stratum3: number;
+    // answer_value_string
+    stratum4: string;
+    // hasSubQuestions
+    stratum7: string;
+    countValue: number;
+    drawerOpen: boolean;
+    countPercent: number;
+    isCopeSurvey: boolean;
+    partcipantCount: number;
+}
+
+interface SurveyRowState {
+    drawerOpen: boolean;
+    subQuestion: object;
+    subAnswers: object;
+    subTitle: string;
+}
+
+
+export const SurveyAnswerRowComponent = (class extends React.Component<SurveyRowProps, SurveyRowState> {
```

Comment:
Nit: simpler to drop `const` and just use class here. Also, no need to export since looks like it's only used in this file.
```suggestion
class SurveyAnswerRowComponent extends React.Component<SurveyRowProps, SurveyRowState> {
```